packages/specs/exceptions/webpack.config.cjs

```
@@ -0,0 +1,4 @@
+module.exports = require("@tsed/webpack-config").create({
+  root: __dirname,
+  name: "exceptions",
```

ESLint

[error] Delete `,`

(prettier/prettier)